packages/@snowpack/plugin-workbox/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@snowpack/plugin-workbox",
+  "version": "0.0.0",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public",
+    "directory": "pkg"
```

Comment:
👍🏻 Glad you added this. This should mean we publish the right directory from Lerna.

Reply:
I'm having trouble getting pika to build separate files on node. need to look into fixing this before using pkg